cmd/main.go

```diff
@@ -74,7 +73,43 @@ func main() {
 		&config.PluginType, "plugin", "csi-plugin", "Type of this driver i.e. controller or node",
 	)
 
+	cmd.PersistentFlags().BoolVar(
+		&config.SetIOLimits, "setiolimits", false,
+		"Whether to set iops, bps rate limit for pods accessing volumes",
+	)
+
+	cmd.PersistentFlags().StringVar(
+		&config.ContainerRuntime, "container-runtime", "containerd",
+		"Whether to set iops, bps rate limit for pods accessing volumes",
+	)
+
+	config.RIopsLimitPerGB = cmd.PersistentFlags().StringSlice(
+		"riops-per-gb", []string{},
+		"Read IOPS per GB limit to use for each volume group prefix, "+
+			"--riops-per-gb=\"vg1-prefix=100,vg2-prefix=200\"",
+	)
+
+	config.WIopsLimitPerGB = cmd.PersistentFlags().StringSlice(
+		"wiops-per-gb", []string{},
+		"Write IOPS per GB limit to use for each volume group prefix, "+
+			"--wiops-per-gb=\"vg1-prefix=100,vg2-prefix=200\"",
+	)
+
+	config.RBpsLimitPerGB = cmd.PersistentFlags().StringSlice(
+		"rbps-per-gb", []string{},
+		"Read BPS per GB limit to use for each volume group prefix, "+
+			"--rbps-per-gb=\"vg1-prefix:100,vg2-prefix:200\"",
+	)
+
+	config.WBpsLimitPerGB = cmd.PersistentFlags().StringSlice(
```

Comment:
we wont be using this, I assume all these command line parameters are optional

Reply:
yes they are optional